### ISSUE TITLE
Falsches errorMessage Format

### ISSUE THREAD

**Issue by Lednerb** [SIWECOS/Version-Scanner#6]
Beim Scan mit Connection Error wird einfalsches Format ausgegeben:

```
{
    "name": "CMSVERSION",
    "version": "1.0.0",
    "hasError": true,
    "errorMessage": "cURL error 28: Resolving timed out after 5516 milliseconds (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)",
    "score": 0,
    "id": 3
  }
```

`errorMessage` muss vom Typ [TranslatableMessage](https://github.com/SIWECOS/siwecos-core-api/tree/ff946d3b1e7a50ce072a6a0c0d1324f6e4dd75e6#translatablemessage-object) sein, um dem Standard zu entsprechen.

**Comment by SniperSister** [comment]
Fixed, bitte prüfen

**Comment by Lednerb** [comment]
@SniperSister 

Ich verstehe den Ansatz, die `EXCEPTION_MESSAGE` mit zu speichern.
Allerdings glaube ich nicht, dass wir diese dem Nutzer übersetzen werden.
Daher könnte man sich den Eintrag der `placeholders` sparen.

@Skeeve was denkst du dazu?

```
{
  "name": "CMSVERSION",
  "version": "1.0.0",
  "hasError": true,
  "errorMessage": {
    "translationStringId": "INTERNAL_ERROR_OCCURED",
    "placeholders": {
      "EXCEPTION_MESSAGE": "cURL error 28: Resolving timed out after 5517 milliseconds (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)"
    }
  },
  "score": 0
}
```

**Comment by Skeeve** [comment]
Also deutsch:
    INTERNAL_ERROR_OCCURED=Es ist ein interner Fehler aufgetreten. Die Meldung lautet: ":EXCEPTION_MESSAGE".

Englisch:
    INTERNAL_ERROR_OCCURED=An internal error occured. The message is:: ":EXCEPTION_MESSAGE".

Oder?

**Comment by Lednerb** [comment]
Korrekt, aber wollen wir dies auch dem Nutzer anzeigen, dass wir einen curl-Error haben?

**Comment by Skeeve** [comment]
Könnte u.U. hilfreich sein.

Alternative: Für jede Exception eine UUID generieren. Mit dieser UUID und einem defnierten "Marker" ins LOG schreiben und melden:

Es ist ein Fehler aufgetreten. Der Administrator wurde benachrichtigt. Fehlercode :UUID

**Comment by Lednerb** [comment]
> Alternative: Für jede Exception eine UUID generieren. Mit dieser UUID und einem defnierten "Marker" ins LOG schreiben und melden:

> Es ist ein Fehler aufgetreten. Der Administrator wurde benachrichtigt. Fehlercode :UUID

> Anhand des Markers können wir solche Exceptions schnell finden und rückfragenden Nutzern helfen. Gegebenenfalls auch angand des Markers monitoren.

Nein, wir beziehen uns hierbei auf die Scanner, die eine externe Entwicklung sind und die wir nicht verpflichten sollten soetwas einzubauen. Es gibt das genannte Nachrichtenformat und dort sollen leicht verständliche Nachrichten für unsere Zielgruppen geliefert werden.

Momentan sehe ich keinen Mehrwert für KMUs darin, bspw. einen englischsprachrigen curl-Fehler auszugeben.
